LibreNMS/Alert/Transport/Jira.php

```diff
@@ -52,7 +52,7 @@ public function contactJira($obj, $opts)
         $password = $opts['password'];
         $prjkey = $opts['prjkey'];
         $issuetype = $opts['issuetype'];
-        $details = 'Librenms alert for: ' . $obj['hostname'];
+        $details = ($obj['title'] != '') ? $obj['title'] : 'Librenms alert for: ' . $obj['hostname'];
```

Comment:
I think using `empty` might a better choice to test the string against.
See https://www.php.net/manual/en/function.empty.php